roles/kubernetes/node-label/tasks/main.yml

```diff
@@ -3,8 +3,6 @@
   uri:
     url: "{{ kube_apiserver_endpoint }}/healthz"
     validate_certs: no
-    client_cert: "{{ kube_apiserver_client_cert }}"
```

Comment:
This should not be removed, when using your external CA you should override the values of these variables if needed.

Reply:
Indeed, you are right that these options don't have an effect but I would rather we remove the `validate_certs: no` and ensure we properly validate the certificates than remove it or make this into a deployer controller setting to allow security conscious deployers to force it to on but leave it off if not needed.